pkg/autoscaler/fake/fake_metric_client.go

```diff
@@ -58,3 +77,24 @@ var StaticMetricClient = MetricClient{
 	StableRPS:         10.0,
 	PanicRPS:          10.0,
 }
+
+// Endpoints is used to create eps
+func Endpoints(count int, svc string) {
```

Comment:
Is both this function and the one in `pkg/autoscaler/scaling/autoscaler_test.go` needed?

Reply:
Thank you @taragu actually i forgot :smile: